### PR TITLE
Adding formatter

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

**Comment by yordis** [comment]
@KamilLelonek how do you make this thing to work with Travis?

Do you have it turn it on by any chance?

**Comment by KamilLelonek** [comment]
It ran automatically.

Z poważaniem,
Kamil Lelonek

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,20 @@
-sudo: false
```
Comment (KamilLelonek) [inline .travis.yml]: Why did you remove that?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
 language: elixir
+
 elixir:
-- 1.5.1
+  - 1.6
+
 otp_release:
-- 20.0
+  - 20.1
```
Comment (KamilLelonek) [inline .travis.yml]: Isn.t `20.2` the newest version?

File: mix.exs [hunk 3]
```diff
@@ -3,30 +3,26 @@ defmodule Exnumerator.Mixfile do
...
   end
 
-  defp apps do
-    []
-  end
-
   defp deps do
     [
-      {:ex_doc, "~> 0.16", only: :dev, runtime: false},
+      {:ex_doc, "~> 0.16", only: :dev, runtime: false}
```
Comment (KamilLelonek) [inline mix.exs:25]: Can you enable comma for the last line?

Comment (yordis) [reply mix.exs:25]: @KamilLelonek I wouldn't, `mix format` did it job. Roll with whatever is the default for the community.
